src/libraries/sendtohelix-wasm.targets

```diff
@@ -194,7 +196,27 @@
         <Command>$(HelixCommand)</Command>
         <Timeout>$(_workItemTimeout)</Timeout>
       </HelixWorkItem>
+
+      <!-- collect the targets files for the tests -->
+      <_TestWithCustomTarget Include="$(TestArchiveTestsDir)\*.helix.targets" />
+      <_TestWithCustomTarget
+          TestName="$([System.IO.Path]::GetFileNameWithoutExtension(%(FileName)))"
+          ArchivePath="$([System.String]::new(%(Identity)).Replace('.helix.targets', '.zip'))" />
+
+      <!-- construct the property name that will contain the target to call for adding
+           helix items -->
+      <_TestWithCustomTarget PropertyName="$([System.String]::new(%(TestName)).Replace('.', '_').Replace('-', '_'))_TargetName" />
```

Comment:
I wonder if this should be more specific than just "TestName_TargetName". Maybe it should say `Helix` in it: ex. "TestName_HelixCustomTargetName".